Source/Advance.cpp

```diff
@@ -28,7 +28,7 @@ ERF::advance(Real time, Real dt, int amr_iteration, int amr_ncycle)
     getFluxReg(level + 1).reset();
   }
 
-  Real dt_new = dt;
+//  Real dt_new = dt;
```

Comment:
I think at some point we're due for another code clean-up. But probably after we have testing in place so we can verify that nothing gets broken.

Reply:
I agree